api/tests/core/bookings/test_repository.py

```diff
@@ -867,6 +867,81 @@ def should_return_validated_bookings_for_offer(self):
             dict(zip(headers, data[1])), beneficiary_2, offer, venue, validated_booking_2, "confirmé", "Non"
         )
 
+    def should_return_validated_bookings_for_offer_with_old_cancelled_booking(self):
+        # Given
+        beneficiary = users_factories.BeneficiaryGrant18Factory(
+            email="beneficiary@example.com", firstName="Ron", lastName="Weasley", postalCode="97300"
+        )
+        beneficiary_2 = users_factories.BeneficiaryGrant18Factory(
+            email="beneficiary2@example.com", firstName="Harry", lastName="Potter", postalCode="97300"
+        )
+        pro = users_factories.ProFactory()
+        offerer = offerers_factories.OffererFactory()
+        offerers_factories.UserOffererFactory(user=pro, offerer=offerer)
+
+        venue = offerers_factories.VenueFactory(managingOfferer=offerer)
```

Comment:
La compréhension s'en trouverait simplifiée en laissant les valeurs/sous-_factories_ par défaut lorsque la surcharge n'est pas nécessaire dans le test, est-ce que ça marcherait tout aussi bien ainsi ?
```suggestion
        beneficiary = users_factories.BeneficiaryGrant18Factory()
        beneficiary_2 = users_factories.BeneficiaryGrant18Factory()
        venue = offerers_factories.VenueFactory()
```

Reply:
Non ça ne marche pas, le teste casse et je n'ai aucune entrée dans le csv